app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -124,4 +124,8 @@ def find_taxonomy
   def allowed_nested_id
     %w(domain_id compute_resource_id subnet_id environment_id hostgroup_id smart_proxy_id user_id medium_id organization_id location_id filter_id)
   end
+
+  def resource_params
+    public_send("#{taxonomy_single}_params".to_sym)
+  end
```

Comment:
This concern is used in both OrganizationsController and LocationsController classes, one with `organization_params` and the other with `location_params`, so this just calls the right one depending on which controller type we're in.